example_configs/presto.yml

```diff
@@ -0,0 +1,48 @@
+lowercaseOutputName: true
```

Comment:
I'm tending towards avoiding these these days.

Reply:
I don't have strong preferences on this. Noticed it had been set to lower in a couple of the other examples and liked the consistent casing is generated so I added it. If you'd like I can drop this.

Reply:
ok, I can drop this.